apps/omar-ui-app/grails-app/assets/javascripts/omar/home/splash.controller.js

```diff
@@ -0,0 +1,81 @@
+(function() {
+  "use strict";
+  angular
+    .module("omarApp")
+    .controller("SplashController", ['$uibModal', '$log', 'stateService', '$stateParams', '$timeout', SplashController]);
+
+  function SplashController($uibModal, $log, stateService, $stateParams, $timeout) {
+    // #################################################################################
+    // AppO2.APP_CONFIG is passed down from the .gsp, and is a global variable.  It
+    // provides access to various client params in application.yml
+    // #################################################################################
+    //console.log('AppO2.APP_CONFIG in SplashController: ', AppO2.APP_CONFIG);
```

Comment:
Commented code. We should consider either always logging this at a particular level, like trace, or removing unused code

Reply:
Yes, this is legacy debugging that was left in to help during development; however, I had always had intentions to do the following:

$logProvider.debugEnabled(AppO2.APP_CONFIG.params.misc.javascriptDebug);

This would be set at a global level for all of our javascript files.  Then in each module we can do:

$log.debug('AppO2.APP_CONFIG in SplashController: ', AppO2.APP_CONFIG);
This allows us to control our debugging dynamically.  Now we can enable/disable javascript debugging via our application.yml or config-repo settings. 

I have this working now locally for the splash.controller.js, but we will need to add a ticket to do this throughout the client-side (UI) modules.